test/goldens/end2end/stream-2.0.1.json_report.md

```diff
@@ -67,14 +67,24 @@ See [package layout](https://dart.dev/tools/pub/package-layout#examples) guideli
 Package not compatible with runtime native-aot
 </summary>
 
-Because of the import of dart:mirrors via the import chain `package:stream/stream.dart` → `package:stream/plugin.dart` → `package:rikulo_commons/mirrors.dart` → `dart:mirrors`
+Because:
+
```

Comment:
nit: Can't we do without this extra whitespace?

Reply:
Done